src/main/java/systems/composable/dropwizard/cassandra/connect/AddressTranslatorFactory.java

```diff
@@ -0,0 +1,26 @@
+package systems.composable.dropwizard.cassandra.connect;
```

Comment:
Please could you rename this package... `connect` is a verb, which doesn't fit nicely with package names. Perhaps something like `connection` or `network`?